src/client/Microsoft.Identity.Client.Desktop/WamExtension.cs

```diff
@@ -27,7 +27,11 @@ internal static void AddSupportForWam(PublicClientApplicationBuilder builder)
             if (DesktopOsHelper.IsWin10OrServerEquivalent())
             {
                 builder.Config.BrokerCreatorFunc =
-                     (uiParent, appConfig, logger) => new Platforms.Features.WamBroker.WamBroker(uiParent, appConfig, logger);                    
+                     (uiParent, appConfig, logger) =>
+                     {
+                         logger.Info("WAM support added.");
```

Comment:
nit: What does "WAM support added." mean? If I saw that in a log message, I may not understand.

Reply:
Modified to "WAM supported OS."